modules/ui.py

```diff
@@ -216,8 +216,14 @@ def __init__(self, is_img2img):
                         outputs=[],
                     )
 
+                    def interrupt_fn():
+                        if shared.state.job_count > 1 and shared.opts.interrupt_after_current:

```

Comment:
About the `self.job_count > 1` condition, I think that's the intuitive behavior that:

- Interrupt right away when generating single image
- Interrupt on next when doing batch
- To interrupt right away in batch, the user could do "Interrupt" and then "Skip".